cluster/compose/localnet/conf/nginx/app-provider.conf

```diff
@@ -5,6 +5,10 @@ server {
     rewrite ^\/(.*) /$1 break;
     proxy_pass http://splice:3${VALIDATOR_ADMIN_API_PORT_SUFFIX}/api/validator;
   }
+  location /registry {
```

Comment:
That doesn't look right to me. /registry is exposed in two places:

1. In scan. That's the part you changed in sv.conf
2. In the validator scan proxy which reads from the scans of all SVs and gives you a BFT read. That's exposed under /api/validator/v0/scan-proxy so the route above covers it already.

So I think you only need to change sv.conf.

I would suggest to extend `LocalNetFrontendIntegrationTest` to make a request to both scan and the validator proxy for user and provider, probably just /registry/metadata/v1/info and test that they work.

Reply:
we'll take a look